task/import.go

```diff
@@ -131,11 +135,9 @@ func prepareImportedAsset(tctx *TaskContext, metadata *FileMetadata, fullPath st
 	}
 	defer importedFile.Close()
 
-	playbackRecordingID, err := Prepare(tctx, metadata.AssetSpec, importedFile)
+	playbackRecordingID, err := Prepare(tctx, metadata.AssetSpec, importedFile, 0.5)
 	if err != nil {
-		glog.Errorf("error preparing imported file assetId=%s err=%q", tctx.OutputAsset.ID, err)
-		// TODO: make these fatal once we're confident about prepare reliability
-		return "", nil
+		glog.Fatalf("error preparing imported file assetId=%s err=%q", tctx.OutputAsset.ID, err)
```

Comment:
These should _not_ be fatal to the whole process. They should only be fatal for the task.

So instead of a fatal log here you should just return the error instead of ignoring it. Otherwise the whole process will crash and every other unrelated task will fail with it as well.